onnxruntime/core/providers/cpu/signal/dft.cc

```diff
@@ -506,7 +506,7 @@ static Status short_time_fourier_transform(OpKernelContext* ctx, bool is_oneside
 
   // Calculate the window size with preference to the window input.
   const auto window_size = window ? window->Shape()[0] : frame_length;
-  ORT_ENFORCE(window_size < signal_size, "Ensure that the dft size is smaller than the signal.");
+  ORT_ENFORCE(window_size <= signal_size, "Ensure that the dft size is smaller than the signal.");
```

Comment:
`is not bigger than the signal`?

Reply:
It is, but in theory could be equal. Saw this edge case, and decided to fix.